sidecar/src/figwheel_sidecar/components/figwheel_server.clj

```diff
@@ -152,25 +158,95 @@
                        trace
                        "</pre>")}))))))
 
-(defn handle-index [handler root]
+(defn checksum-file
+  "Generated from Pandect"
+  ;; TODO: full attribution^^
+  [^File file]
```

Comment:
We could look at modifying this so that the file is only read once while serving it, rather than once here, and once in the response.